sql/storm-sql-runtime/src/test/org/apache/storm/sql/TestUtils.java

```diff
@@ -41,7 +26,46 @@
 import java.util.Map;
 import java.util.PriorityQueue;
 
+import org.apache.storm.spout.SpoutOutputCollector;
+import org.apache.storm.sql.runtime.ISqlStreamsDataSource;
+import org.apache.storm.streams.Pair;
+import org.apache.storm.task.OutputCollector;
+import org.apache.storm.task.TopologyContext;
+import org.apache.storm.topology.IRichBolt;
+import org.apache.storm.topology.IRichSpout;
+import org.apache.storm.topology.OutputFieldsDeclarer;
+import org.apache.storm.topology.base.BaseRichBolt;
+import org.apache.storm.topology.base.BaseRichSpout;
+import org.apache.storm.tuple.Fields;
+import org.apache.storm.tuple.Tuple;
+import org.apache.storm.tuple.Values;
+import org.junit.rules.ExternalResource;
+
 public class TestUtils {
+  public static final ExternalResource mockInsertBoltValueResource = new ExternalResource() {
+    @Override
+    protected void before() throws Throwable {
+      MockInsertBolt.getCollectedValues().clear();
+    }
+
+    @Override
+    protected void after() {
+      // no-op
```

Comment:
Nit: I think you can leave out this method entirely

Reply:
Will fix.